hamocc/mo_intfcblom.F90

```diff
@@ -482,17 +483,17 @@ subroutine hamocc2blom(m,n,mm,nn)
         km=k+mms
         kn=k+nns
         do j=1,jj
-           do l=1,isp(j)
-              do i=max(1,ifp(j,l)),min(ii,ilp(j,l))              ! time smoothing (analog to tmsmt2.F)
-                 sedlay2(i,j,km,:) = wts1*sedlay2(i,j,km,:)   &  ! mid timelevel
-                      + wts2*sedlay2(i,j,kn,:)                &  ! old timelevel
-                      + wts2*sedlay(i,j,k,:)                     ! new timelevel
-                 powtra2(i,j,km,:) = wts1*powtra2(i,j,km,:)   &
-                      + wts2*powtra2(i,j,kn,:)                &
-                      + wts2*powtra(i,j,k,:)
-                 burial2(i,j,m,:)  = wts1*burial2(i,j,m,:)    &
-                      + wts2*burial2(i,j,n,:)                 &
-                      + wts2*burial(i,j,:)
+        do l=1,isp(j)
+        do i=max(1,ifp(j,l)),min(ii,ilp(j,l))              ! time smoothing (analog to tmsmt2.F)
+           sedlay2(i,j,km,:) = wts1*sedlay2(i,j,km,:)   &  ! mid timelevel
+                             + wts2*sedlay2(i,j,kn,:)   &  ! old timelevel
+                             + wts2*sedlay(i,j,k,:)        ! new timelevel
+           powtra2(i,j,km,:) = wts1*powtra2(i,j,km,:)   &
+                             + wts2*powtra2(i,j,kn,:)   &
+                             + wts2*powtra(i,j,k,:)
+           burial2(i,j,m,:)  = wts1*burial2(i,j,m,:)    &
+                             + wts2*burial2(i,j,n,:)    &
+                             + wts2*burial(i,j,:)
```

Comment:
@mvertens
this is a good example of what I mean with the vertical alignment. I guess these changes would be reverted by an auto-indent?

Reply:
@JorgSchwinger - yes it is. Thanks for the clarification. I agree that these changes would be reverted. 
Not sure if we could automate that. Maybe a meeting to discuss all of these issues would be good.

Reply:
Yes, at least in my experience, auto-indent will normally indent by a fixed number of spaces for line continuation.